### ISSUE TITLE
How to set image full screen width and height

### ISSUE THREAD

**Issue by sergiobr** [TangoAgency/material-intro-screen#109]
Hi,

I'm showing a serie of images that should be fiting the whole screen. It fills the height but not the width, staying with spaces on both sides.

Here is my XML:
`<?xml version="1.0" encoding="utf-8"?>
<agency.tango.materialintroscreen.parallax.ParallaxLinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="fill_parent"
    android:layout_height="fill_parent"
    android:background="@color/primary_bar"
    android:orientation="vertical"
    android:gravity="center">

    <ImageView
        android:id="@+id/slide1"
        android:layout_width="fill_parent"
        android:layout_height="fill_parent"
        android:scaleType="fitCenter"
        android:src="@drawable/slide1" />

</agency.tango.materialintroscreen.parallax.ParallaxLinearLayout>`

**Comment by sergiobr** [comment]
Please, anybody?

**Comment by TrentPierce** [comment]
Did you ever have any luck with this? I cant get it to work either. I just need the image to fill_parent. Unfortunately, it looks like the only solution is to change libraries.

**Comment by bezmian** [comment]
Hi, can you provide more info about that? Maybe some screenshots how it looks. Also keep in mind that fill_parent is deprecated and renamed to MATCH_PARENT in API Level 8 (https://developer.android.com/reference/android/view/ViewGroup.LayoutParams.html).
My first idea is to use `android:scaleType` in your xml. Here are examples:
![image](https://user-images.githubusercontent.com/17637583/29766224-554a233a-8bde-11e7-9333-231fa3469128.png)
![image](https://user-images.githubusercontent.com/17637583/29766232-5d0f908c-8bde-11e7-92a1-a19d4bd2c8a9.png)

**Comment by TrentPierce** [comment]
Yea, sorry, I was in a bit of a hurry when I commented. I went ahead and forked this library and made the changes I needed. Here's how it looks now. In my fork there is information on how to use the library through jitpack. 

![resized](https://user-images.githubusercontent.com/1277122/29772506-be5c88ce-8bbe-11e7-956e-05b37fc9e435.jpg)

It's working perfectly for me now. As far as changes go, I just removed the margins and padding on the images to fit my needs. If you use lots of text in the text views it'll probably run out of room. It's great if you're using images with text already in them. 

[My Fork](https://www.github.com/TrentPierce/material-intro-screen)
